server/lib/utils/base.ts

```diff
@@ -199,3 +199,7 @@ export function invert (obj: Record<string | number, string | number>) {
   }
   return invertedObj
 }
+
+export function getOptionalValue <T extends object, K extends keyof T> (obj: T, attribute: K) {
```

Comment:
[naming is hard] I'm confused by the `get` part (suggesting something more far-fetched than what it is (ie. a db request)) and by the `optional` part (isn't it always optional if the key is not defined in the object?)

Suggestions by order of preference:
- property : which gives a lodash/underscore flavour
- propertyValue: less generic than above
- objectValue
- objectKeyValue

I think not having a verb would be preferable to indicate that it's a small function

Reply:
Yep, I knew this function's name would come up ^^

The `get` part is actually mimicking the likes of [lodash](https://lodash.com/docs/4.17.15#get)/underscore/backbone to access an object attribute's value. I still prefer it to the proposed alternatives. I think the `Optional` part of the name, or an equivalent signifier, should stay, as that's the purpose of that function: to not get a type error when trying to access an optional object attribute

Reply:
Discussion made obsolete by c6a5669 ^^'